### DISCUSSION TITLE
Export to CSV from command line?

### DISCUSSION THREAD

**Discussion by aliddell** [VerySleepy/verysleepy#122] [Q&A]
I know that it's [possible](https://github.com/VerySleepy/verysleepy/wiki/Command-Line-Usage) to export the output of a VerySleepy session to a .sleepy file, but is it possible to then convert that file to a CSV programmatically?

![image](https://github.com/user-attachments/assets/faf8601c-5f08-434e-b458-402d1ad499c0)

I don't see any such option here ☝️.

What I'm interested in doing is batch profiling my program with different data sizes, but I have to manually open up each .sleepy file and export to CSV. It's a bit tedious.

**Comment by CyberShadow** [comment]
The `.sleepy` file is pretty easy to parse programmatically - it is a ZIP file with text files inside. Perhaps you may find it easier to just do that?

**Reply by aliddell** [reply]
Is there a spec for that somewhere, or do I need to read the source?

**Reply by CyberShadow** [reply]
No spec document. Hopefully most of it should be self-evident from just looking at a sample `.sleepy` file, but if anything isn't clear, the source code is indeed your best bet for additional clarity.
